build.gradle

```diff
@@ -353,23 +353,26 @@ subprojects {
 }
 
 
+/* LOCAL DEVELOPMENT HELPER TASKS */
```

Comment:
How about `/* BEGIN: LOCAL ...`?

Reply:
I've been kinda using this syntax throughout the file. Can change later, but this is self consistent for now.